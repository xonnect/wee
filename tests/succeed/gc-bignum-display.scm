;; bug: after GC, bignum display becomes odd

(displayln (* 21435 2141241))
(displayln (* 21435 2141241))
(displayln (* 21435 2141241))
(displayln (* 21435 2141241))
(displayln (* 21435 2141241))
(displayln (* 21435 2141241))
(displayln (* 21435 2141241))
(displayln (* 21435 2141241))
(displayln (* 21435 2141241))
(displayln (* 21435 2141241))
(displayln (* 21435 2141241))
(displayln (* 21435 2141241))
