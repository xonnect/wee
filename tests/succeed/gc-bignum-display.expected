45897500835
45897500835
45897500835
45897500835
45897500835
45897500835
45897500835
45897500835
45897500835
45897500835
45897500835
45897500835
